emackinnon1/overlook.github.io
# Feature/iteration4

#### What's this PR do?
This PR allows the manager to delete a upcoming booking for a user, but the list needs to update afterwards.
#### Where should the reviewer start (files changed)?
index.js, dom.js
#### How should this be manually tested?
Sign in as manager, search a user by name, click the cancel booking and refresh the page to see the update list 
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
